### PR TITLE
#165273592  User can like and dislike comment

### PR DESCRIPTION
**Pull request by nadralia**
### What does this PR do?
This pull request enables one to like and dislike a specific comment

### Description of tasks to be completed?
An authenticated user can like/dislike a specific comment
User cannot like their own comment

### How should this be tested?

- Clone the Github repository - `git clone https://github.com/andela/ah-backend-dojo.git`
- Navigate into the repository -` cd ah-backend-dojo`
- Create a virtual environment - `python3 -m venv [virtual_environment_name]`
- Activate the virtual environment - macOS: `source [virtual_environment_name]/bin/activate`
- Install the dependencies from the requirements file - `pip install -r requirements.txt`
- Create a .env file, copy and paste in environment variables setup in the .env_sample file, in the blank quotes add your PostgreSQL database credentials.
- Make the migrations - `python manage.py makemigrations`
- Migrate -` python manage.py migrate`
- Run the server - `python manage.py runserver`
- Run your postman application

#### Like a comment
Create an article and obtain its slug
Add a comment to that article and get its id
Replace the 'slug' and 'id' in POST /api/articles/:slug/comments/:id/like/ and send a POST request

#### Dislike a comment
Create an article and obtain its slug
Add a comment to that article and get its id
Replace the 'slug' and 'id' in /api/articles/:slug/comments/:id/dislike/ and send a DELETE request

[Finishes #165273592]

### CODE REVIEW

File: authors/apps/comments/tests/test_like_dislike_comment.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            "You have liked this comment",
+        )
+        response2 = self.client.post(
+            f"/api/articles/{self.article.slug}/comments/1/like",
+            content_type="application/json",
+        )
+        self.assertEqual(response2.status_code, 200)
+        self.assertEqual(
+            response2.data["message"],
+            "You have already like the comment",
```
Comment (bisonlou) [inline authors/apps/comments/tests/test_like_dislike_comment.py]: Wonderful work @nadralia. Just mind the grammar here.

File: authors/apps/comments/views.py [hunk 2]
```diff
@@ -65,3 +67,72 @@ def destroy(self, request, *args, **kwargs):
...
+        """Like a comment """
+        current_user = request.user
+        user = Profile.objects.get(user=current_user)
+        user_id = user.id
+        comment_id = kwargs['pk']
+        liked_comment = CommentLikeDislike.objects.filter(like=True)
+        liked_by = CommentLikeDislike.objects.filter(user_id=user_id)
+        if liked_comment and liked_by:
+            return Response(
+                    data={"message": "You have already like the comment"},
```
Comment (bisonlou) [inline authors/apps/comments/views.py]: grammar please. how about "You already like this comment"

**Review by bisonlou — APPROVED** [review]
Just check the grammar and this looks good to me.

File: authors/apps/comments/views.py (hunk 2 above)
Comment (nadralia) [reply authors/apps/comments/views.py]: Let me correct that

**Review by bisonlou — APPROVED** [review]
Just check the grammar and this looks good to me.

**Review by JamesMudidi — APPROVED** [review]
@nadralia Nice work bro.

**Review by kalsmic — APPROVED** [review]
well done @nadralia, I like how simple you have kept the comment likes, its either a thumbs up or nothing at all. kudos

**Review by ZackAtama — APPROVED** [review]
Awesome work @nadralia.

**Pull request merged**